src/themes/dark.tsx

```diff
@@ -19,6 +19,12 @@ const darkTheme = {
   heroImageBackgroundColor: "#2f363d !important",
   subtitleColor: colors.gray[4],
   mapBackgroundColors: {
+    "New Queen Street": "#27dfdf",
+    Esperança: "#d3b8a7",
+    Colosseo:"#e79b68",
```

Comment:
```suggestion
    Colosseo: "#e79b68",
```
![image](https://user-images.githubusercontent.com/82317/201140282-75902723-121c-442e-acfe-7b482a2763db.png)